src/renderer/pages/project/script_editor/beat_style.vue

```
@@ -5,15 +5,37 @@
+    </CollapsibleContent>
   </Collapsible>
 </template>
 
 <script setup lang="ts">
 import { ref } from "vue";
 import { Collapsible, CollapsibleContent, CollapsibleTrigger } from "@/components/ui/collapsible";
-import { Button } from "@/components/ui/button";
+import { Button } from "@/components/ui";
+
+interface Props {
+  beat: MulmoBeat;
```

ESLint

[error] 'MulmoBeat' is not defined.

(no-undef)